Harivigneshkumar/Contact Form 1/index.html

```diff
@@ -0,0 +1,276 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>index</title>
+    <style>
+        table,th,td {
```

Comment:
Move this CSS code to separate Style file and provide link in the HTML page.
Indent the CSS code properly